.github/workflows/publish_image.yml

```diff
@@ -1,7 +1,10 @@
 name: Publish docker image
 
 on:
+  workflow_dispatch:
   push:
+    branches:    
+      - 'perf/optimize-suboptimal-code'
```

Comment:
I suppose we need to remove this to merge?

Reply:
I will remove it after we deploy and test it on staging

Reply:
Why is that? If we are going to merge to master the image should come from master. Unless you are going to deploy before merging.

Reply:
Damian created staging env on AWS, so we will use that for test before merging it to the master.